app/src/main/res/layout/activity_pinch_to_zoom.xml

```diff
@@ -1,33 +1,46 @@
-        xmlns:android="http://schemas.android.com/apk/res/android"
-        xmlns:tools="http://schemas.android.com/tools"
-        xmlns:app="http://schemas.android.com/apk/res-auto"
-        android:layout_width="match_parent"
-        android:layout_height="match_parent"
-        android:background="@color/black_80_opacity"
-        tools:context="br.com.concrete.tentacle.features.library.loan.PinchToZoomActivity">
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:background="@color/black_80_opacity"
+    tools:context="br.com.concrete.tentacle.features.library.loan.PinchToZoomActivity">
+
+    <view
+        android:id="@+id/youtube_player"
+        class="com.google.android.youtube.player.YouTubePlayerView"
+        android:layout_width="@dimen/dp_0"
+        android:layout_height="@dimen/dp_0"
+        android:layout_margin="@dimen/dp_8"
+        android:visibility="gone"
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintDimensionRatio="16:9"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent" />
+
 
     <com.github.chrisbanes.photoview.PhotoView
-            android:id="@+id/photoView"
-            android:layout_width="@dimen/dp_0"
-            android:layout_height="@dimen/dp_0"
-            android:layout_margin="@dimen/dp_8"
-            app:layout_constraintDimensionRatio="16:9"
-            app:layout_constraintTop_toTopOf="parent"
-            app:layout_constraintBottom_toBottomOf="parent"
-            app:layout_constraintStart_toStartOf="parent"
-            app:layout_constraintEnd_toEndOf="parent"/>
+        android:id="@+id/photoView"
+        android:layout_width="@dimen/dp_0"
+        android:layout_height="@dimen/dp_0"
+        android:layout_margin="@dimen/dp_8"
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintDimensionRatio="16:9"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent" />
 
     <ImageView
-            android:id="@+id/btClose"
-            android:src="@drawable/ic_close"
-            android:layout_width="@dimen/dp_25"
-            android:layout_height="@dimen/dp_25"
-            android:layout_marginTop="@dimen/dp_25"
-            android:layout_marginEnd="@dimen/dp_25"
-            android:background="@drawable/circle_bg_close"
-            app:layout_constraintRight_toRightOf="parent"
-            app:layout_constraintTop_toTopOf="parent"/>
+        android:id="@+id/btClose"
+        android:layout_width="@dimen/dp_25"
```

Comment:
Hoje eu não usaria esse tipo de dimen. Tem dois pontos aqui.
1 - Não é recomendado o uso de tamanhos impares
2 - Não tem diferença colocar 25dp hardcoded ou @dimen/dp_25

A melhor abordagem nesse caso seria a criação de padrões. Exemplo:

default = 8dp
medium = 16dp
large = 32dp

os nomes são apenas sugestões, vai depender do alinhamento dom designer e dev team.
A vantagem disso é que se no futuro o meu default não for mais 8dp eu simplesmente vou mudar o valor dele no dimens e refletir em todo o app. Agora imagina se o preciso dizer que onde tem dp_25 agora deve ter do_30. Ficaria muito ruim mudar o valor do dp_35 para 30 né. Ou muito trabalhoso ter que ficar  trocando em todo o código. Sacou??

Reply:
saquei, então devo rever o código feito antes e fazer alterações que o melhore, certo?

Reply:
Não, por enquanto não. Podemos pensar mais para a frente em refatorar tudo para um módulo de design por exemplo. Mas por hora usa assim mesmo. Só comentei para ficar no seu radar.